pep-9999.rst

```diff
@@ -333,18 +333,20 @@ type of ``x`` be ``str`` or ``Literal["blue"]``?
 This PEP does not require any particular strategy for cases like this,
 apart from requiring that backwards compatibility is maintained.
 
-For example, one valid strategy might be to analyze the surrounding code
-to deduce the appropriate type for ``x``. Another valid strategy would
-be to assume that ``x`` is always of type ``str`` and require users to
-manually annotate it if they want it to have a Literal type. A third strategy
-might be to use a hybrid of the previous two.
-
-However, type checkers should take care to avoid being too over-zealous
-when performing inference. For example, one strategy that does *not* work is
-always assuming expressions are Literal types. This naive strategy would cause
-programs like the following to start failing when they previously did not::
-
-    # If we infer 'var' has type Literal[3]
+For example, one simple strategy for meeting this requirement would be
+to always assume expressions are *not* Literal types unless they are
+explicitly annotated otherwise. A type checker using this strategy would
+always infer that ``x`` is of type ``str`` in the above example.
+
+If type checkers choose to use more sophisticated inference strategies
+(e.g. using the surrounding context to infer the appropriate type for ``x``),
```

Comment:
This seems much better, but I think still doesn't need this e.g. (as I think it begs more questions about what that context means).
Not super hung up on it though.

Reply:
Sure -- I removed the e.g. I'm trying to find ways to make this doc shorter anyways.